### PR TITLE
Make the conformance tool work with Functions Framework for Node.js

### PR DESCRIPTION
**Pull request by hdp617**

### CODE REVIEW

File: client/validate.go [hunk 1]
```diff
@@ -16,7 +29,7 @@ const (
 
 // The HTTP function should copy the contents of the request into the response.
 func validateHTTP(url string) error {
-	req := "PASS"
+	req := `{"res":"PASS"}`
```
Comment (juliehockett) [inline client/validate.go:32]: Please update the readme :)

Comment (hdp617) [reply client/validate.go:32]: Done.

**Comment by grant** [comment]
@hdp617 Is this PR ready?

**Comment by hdp617** [comment]
Oops, sorry I forgot to follow up. This PR is ready. Could you merge/close it? I don’t have push access.

**Comment by grant** [comment]
Fixed access to FF org.

**Pull request merged**